pkg/controller/vitesskeyspace/reconcile_shards.go

```diff
@@ -141,6 +141,7 @@ func newVitessShard(key client.ObjectKey, vtk *planetscalev2.VitessKeyspace, par
 			BackupEngine:           vtk.Spec.BackupEngine,
 			ExtraVitessFlags:       vtk.Spec.ExtraVitessFlags,
 			TopologyReconciliation: vtk.Spec.TopologyReconciliation,
+			MaxConditionDurations:  make(map[planetscalev2.VitessShardConditionType]metav1.Duration),
```

Comment:
Nit: I think it reads better as `ConditionMaxDurations`

Reply:
I don't feel strongly either way so I'm happy to change it.

Reply:
Updated: https://github.com/planetscale/vitess-operator/pull/41/commits/f9dc9ba9dab7e0588af7a5bea78c9759dabbd61d.